cola/gitcmds.py

```diff
@@ -658,6 +658,28 @@ def diff_upstream(context, head):
     return diff_filenames(context, base, tracked)
 
 
+def list_submodule(context):
+    """Return submodules in the format(state, sha1, path, describe)"""
+    git = context.git
+    SHA1_LENGTH = 40
+    status, data, err = git.submodule()
```

Comment:
this might be slightly more explicit by saying `git.submodule('status')` instead